src/ImageSharp/Common/Helpers/SimdUtils.HwIntrinsics.cs

```diff
@@ -577,6 +577,24 @@ public static class HwIntrinsics
                 }
             }
 
+            /// <summary>
+            /// Scales 8x8 matrix to 4x2 using 2x2 average
+            /// </summary>
+            /// <param name="v">Input matrix consisting of 4 256bit vectors, first row: (v[0], v[2]), second row: (v[1], v[3])</param>
+            /// <returns>256bit vector containing upper and lower scaled parts of the input matrix</returns>
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            public static Vector256<float> Scale16x2_8x1(ReadOnlySpan<Vector256<float>> v)
```

Comment:
Does this method have a use case outside of the jpeg encoder? If not, it shouldn't be in this class. `SimdUtils` is supposed to be for reusable utility methods. 

Anything public should also have explicit `IsSupported` checks.

Reply:
It's not, moved to jpeg pixel converter. Added `Debug` check for Avx2.